packages/server/src/controllers/UsersController.js

```diff
@@ -4,7 +4,135 @@ import UsersValidator from '../validators/UsersValidator'
 
 const { User, UserAddress } = Models
 
+// temp dummy models
+let Users = { }
+let UserEmails = { }
+let UserSessions = { }
+
+
 export default {
+  async details(req, res) {
+    let body = {
+      "emailExists": false,
+      "emailVerified": false,
+      "addressVerified": false,
+      "notificationsEnabled": false
+    }
+    Object.assign(body, Users[req.params.address])
+    res.send(body)
+  },
+
+
+  sessions: {
+    async create(req, res) {
+      if (!(req.params.address in Users)) {
+        Users[req.params.address] = {}
+      }
+      Users[req.params.address]['addressVerified'] = true
+      if (!(req.params.address in UserSessions)) {
+        UserSessions[req.params.address] = {}
+      }
+      UserSessions[req.params.address][req.session.id] = true
+      let body = {
+        "authenticated": true
+      }
+      res.send(body)
+    },
+
+    async authenticate(req,res,next) {
+      if (
+        !(req.params.address in UserSessions) ||
+        !(req.session.id in UserSessions[req.params.address])
+      ) {
+        // exception for new emails from anj
+        if (req.path.endsWith("/email") && !(req.params.address in Users)) {
+          return next()
+        }
```

Comment:
I think skipping particular paths here is a bit weird, we can do something like the `asyncMiddleware` for authenticated routes, see my example [here](https://github.com/aragonone/court-backend/pull/133/files?file-filters%5B%5D=.js&file-filters%5B%5D=.sample#diff-d8751b0865f851a3e04a955c3c19aa33R46-R49)

Reply:
Yes I agree but we need to check this special case somehow wherever it is.
I was thinking why do we even allow this anonymous situation? Let's just ask every new subscriber to authenticate and verify same as for the client (we will only treat previously subscribed jurors in a special way).

Reply:
Removed this part, let's just reuse your old `POST` `/users` endpoint for anonymous requests.